pkg/core/storelimit/sliding_window.go

```diff
@@ -62,13 +87,20 @@ func (s *SlidingWindows) Reset(cap float64, typ Type) {
 	}
 }
 
+// GetCap returns the capacity of the sliding windows.
+func (s *SlidingWindows) GetCap() int64 {
+	s.mu.RLock()
+	defer s.mu.RUnlock()
+	return s.windows[0].capacity
+}
+
 // GetUsed returns the used size in the sliding windows.
-func (s *SlidingWindows) GetUsed() int64 {
+func (s *SlidingWindows) GetUsed() []int64 {
```

Comment:
`GetUsed ` is only used in metrics and test?

Reply:
yes